portal-web/docroot/html/common/themes/top_messages.jsp

```diff
@@ -28,31 +28,38 @@ String message = (String)PortalMessages.get(request, PortalMessages.KEY_MESSAGE)
 
 if (Validator.isNotNull(jspPath) || Validator.isNotNull(message)) {
 	String cssClass = GetterUtil.getString(PortalMessages.get(request, PortalMessages.KEY_CSS_CLASS), "alert-info");
+	String displayType = GetterUtil.getString(PortalMessages.get(request, PortalMessages.KEY_DISPLAY_TYPE), "success");
```

Comment:
Default value we set above in the `request` was `info` but now we default to `success`... I'm still failing to see where this is being set elsewhere in the code, so I'd still be reluctant to get this in.

@diegonvs, could you explain how is this supposed to be used from the outside?

Reply:
Yes, default to success don't make any sense(considering https://github.com/liferay-frontend/liferay-portal/pull/39#discussion_r443570692), changing.